### DISCUSSION TITLE
[Section 3.12 - Alterando e destruindo] Arquivos do Bucket

### DISCUSSION THREAD

**Discussion by vladimiremi** [chgasparoto/curso-aws-com-terraform#140] [Q&A]
Na aula você mostra a criação e a deleção de um recurso. Partindo em um cenário onde eu tenha vários arquivos no meu bucket, esse recurso deletado vai apagar também todos os arquivos que eu já tinha salvo?

**Comment by chgasparoto** [comment]
@vladimiremi Resumindo, não. 

Se os arquivos no bucket não foram criados pelo terraform o s3 vai retornar um erro na hora de deletar o bucket e não vai deixar deletar enquanto tiver arquivos não gerenciados pelo terraform lá dentro.
